[CUBRIDMAN-224] Manual for correlated subquery result cache (#526)
CUBRID/cubrid-manual: en/admin/config.rst

|                               +-------------------------------------+-------------------------+---------+----------+--------------------------------+-----------------------+
|                               | max_hash_list_scan_size             | server parameter        |         | byte     | 8,388,608(8M)                  |                       |
|                               +-------------------------------------+-------------------------+---------+----------+--------------------------------+-----------------------+

> |                               | max_subquery_cache_size             | server parameter        |         | byte     | 2,097,152(2M)                  |                       |

Reviewer: ```suggestion
|                               | max_subquery_cache_size             | server parameter        |         | byte     | 2,097,152(2M)                  | DBA only              |
```

+--------------------------------+--------+---------------------------+---------------------------+---------------------------+
| thread_stacksize               | byte   | 1,048,576                 | 65,536                    |                           |
+--------------------------------+--------+---------------------------+---------------------------+---------------------------+

> | max_subquery_cache_size        | byte   | 2,097,152(2M)             | 0                         | 16,777,216(16M)           |

Reviewer: 알파벳 순서로 작성되어 있는 것 같습니다.
max_subquery_cache_size를 마지막 위치가 아닌 적절한 위치로 옮겨야 할 것 같아요.

변경 시, 이곳 말고 다른 곳들도 같이 변경해주세요.

Author: 코드 리뷰 적용했습니다. 확인 부탁드립니다.

    If this parameter is set to 0 or If :ref:`NO_HASH_LIST_SCAN <no-hash-list-scan>` hint is specified, hash list scan will not be used.

**max_subquery_cache_size**

>     **max_subquery_cache_size** parameter is used to set the size of the subquery cache (correlated). You can set a unit as B, K or M, which stand for bytes, kilobytes (KB),and megabytes (MB) respectively. If you omit the unit, bytes will be applied. The default value is **2,097,152** (2M) bytes, the minimum value is **0**, and the maximum value is **16,777,216** (16M) bytes. The subquery cache is allocated for the number of subqueries in a query and is deallocated when the main query is completed. 

Reviewer 2: ```suggestion
    **max_subquery_cache_size** is a parameter used to set the size of the subquery cache (correlated). You can set a unit as B, K or M, which stand for bytes, kilobytes (KB), and megabytes (MB), respectively. If you omit the unit, bytes will be applied. The default value is **2,097,152** (2M) bytes, the minimum value is **0**, and the maximum value is **16,777,216** (16M) bytes. The subquery cache is allocated for the number of subqueries in a query and is deallocated when the main query is completed. 
```

>     If max_subquery_cache_size is set to 0, the :ref:`NO_SUBQUERY_CACHE <correlated-subquery-cache>` hint is specified, or there is insufficient storage space, the subquery cache will not be used.

Reviewer: ```suggestion
    If max_subquery_cache_size is set to 0, the :ref:`NO_SUBQUERY_CACHE <correlated-subquery-cache>` hint is specified, or there is insufficient storage space, the subquery cache is not used.
```

>     If max_subquery_cache_size is set to 0, the :ref:`NO_SUBQUERY_CACHE <correlated-subquery-cache>` hint is specified, or there is insufficient storage space, the subquery cache is not used.

Reviewer 3: ```suggestion
    If max_subquery_cache_size is set to 0, the :ref:`NO_SUBQUERY_CACHE <correlated-subquery-cache>` hint is specified, or there is insufficient memory space, the subquery cache optimization is not enabled.
```

    **thread_stacksize** is a parameter to configure the stack size of a thread. The default value is **1048576** bytes. The value of the **thread_stacksize** parameter must not exceed the stack size allowed by the operating system.

**max_subquery_cache_size**

    **max_subquery_cache_size** is a parameter to configure the size of subquery cache (correlated). 
    The default value is **2097152** bytes.

>     If there is insufficient storage space, no further subquery caching will be possible.

Reviewer: 일관성을 위해 다른 파라미터들처럼 다음의 언급이 필요합니다.
1. 최소값, 최대값
2. NO_SUBQUERY_CACHE 힌트
3. 0으로 설정되었을 경우의 동작
4. `sort_buffer_size`의 경우 "서버는 각 클라이언트의 정렬 요청마다 하나의 정렬 버퍼를 할당하며, 정렬을 완료한 후에는 할당되었던 버퍼 메모리를 해제한다."라고 되어 있습니다. `max_subquery_cache_size`도 이와 관련해서 어떻게 동작하는지 짧은 설명이 필요합니다. 예를 들어, 질의의 서브 쿼리 개수만큼 할당되며, 서브 쿼리가 종료될 때 해제되는지? 또는 질의가 종료될 때 한 번에 해제되는지? 등의 설명이 필요합니다.

Author: 코드 리뷰 적용했습니다. 확인 부탁드립니다.
